web/index.ts

```diff
@@ -299,24 +282,22 @@ const App = (_: any, state: AppState, setState: SetState) => {
                         }),
                         H('div',
                             { className: 'field-flex' },
-                            H(Dropdown, {
-                                options: widthOptions,
+                            H(TextInput, {
                                 value: widths[0],
-                                small: true,
-                                onchange: (val: string) =>  {
+                                small:true,
```

Comment:
We need a `placeholder: 'width'` so its clear what this input is supposed to mean

Reply:
```suggestion
                                small: true,
```